elastic/kibana
# [Entity Analytics] Ensuring definition transforms are managed

## Summary

This PR adds the `managed` flag to the entity definitions so that the generated transforms are tagged as such.

### How to test

1. Add `EntityStoreEnabled` feature flag to your `kibana.dev.yml`
2. Call `POST kbn:/api/entity_store/engines/<user | host>/init` with an empty (`{}`) body.
3. Go to `Stack Management > Transforms` and verify that the 2 new transforms are tagged as managed

Implements https://github.com/elastic/security-team/issues/10608
